09-Time-Series-Data.ipynb

```diff
@@ -0,0 +1,794 @@
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "plt.figure(1, figsize=(10,5))\n",
+    "plt.plot(lyra_lightcurve.data.index, lyra_lightcurve.data['CHANNEL3'], color='blue', label='Al filter')\n",
+    "plt.plot(lyra_lightcurve.data.index, lyra_lightcurve.data['CHANNEL4'], color='red', label='Zr filter')\n",
+    "plt.ylabel('Flux (Wm^2)')\n",
+    "plt.show()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Analysing Lightcurve data\n",
+    "\n",
+    "We can asses the degree to which the lyra curves correlate with each other:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "cross_correlation = np.correlate(lyra_lightcurve.data['CHANNEL3'],\n",
+    "                                 lyra_lightcurve.data['CHANNEL4'])\n",
+    "print(cross_correlation)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pandas\n",
+    "\n",
+    "In its own words Pandas is a Python package providing fast, flexible, and expressive data structures designed to make working with “relational” or “labeled” data both easy and intuitive. Pandas has two forms of structures, 1D series and 2D dataframe. It also has its own functions associated with it.\n",
+    "\n",
+    "It is also amazing.\n",
+    "\n",
+    "Lightcurve uses these in built Pandas functions, so we can find out things like the maximum of curves:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# max time argument taken from long and short GOES channels\n",
+    "max_t_goes_long = goes_lightcurve.data['xrsb'].idxmax()\n",
+    "max_t_goes_short = goes_lightcurve.data['xrsa'].idxmax()\n",
+    "\n",
+    "# max_time argument taken from channel 3 & 4 LYRA channe;s\n",
+    "max_t_lyra_al = lyra_lightcurve.data['CHANNEL3'].idxmax()\n",
+    "max_t_lyra_zr = lyra_lightcurve.data['CHANNEL4'].idxmax()\n",
+    "\n",
+    "print('GOES long: ', max_t_goes_long)\n",
+    "print('GOES short: ', max_t_goes_short)\n",
+    "print('LYRA Al: ', max_t_lyra_al)\n",
+    "print('LYRA Zr: ', max_t_lyra_zr)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "So lets plot them on the graph"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# create figure with raw curves\n",
+    "plt.figure(1, figsize=(10,5))\n",
+    "plt.plot(lyra_lightcurve.data.index, lyra_lightcurve.data['CHANNEL3'], color='blue', label='Al filter')\n",
+    "plt.plot(lyra_lightcurve.data.index, lyra_lightcurve.data['CHANNEL4'], color='red', label='Zr filter')\n",
+    "plt.ylabel('Flux (Wm^2)')\n",
```

Comment:
If the lightcurve includes the unit, then I would create the label from it (I don't know whether it's somewhere), eg:

```python
'Flux ({})'.format(lyra_lightcurve.unit)
```